doc/install.rst

```diff
@@ -79,7 +80,7 @@ First, we must configure conda to get packages from the
 Now we can create a new conda environment with Python and all our dependencies installed
 (we'll call it ``pygmt`` but you can change it to whatever you want)::
 
-     conda create --name pygmt python=3.6 pip numpy pandas xarray packaging gmt=6.0.0
+     conda create --name pygmt python=3.6 pip numpy pandas xarray netcdf4 packaging gmt=6.0.0
```

Comment:
Once #430 is merged, would it be better to just instruct users to `conda install pygmt` since it'll pull in all the side dependencies as well?

Reply:
Yes, perhaps we should merge https://github.com/conda-forge/pygmt-feedstock/pull/2 first, then instruct users to install pygmt with:
```
conda create --name pygmt pygmt
```